Good-Bids/goodbids
# [#88] Bid Restrictions

# Summary

This PR applies a few adjustments to the Bid Product:

* Set the stock level to 1
* Reset the stock level after a bid is purchased
* Redirect to Checkout after Adding a Bid to Cart
* Restrict access to the Bid Product page
* Enable Virtual and Sold Individually on the Bid Product.

## Issues

* #88 

## Testing Instructions

1. Create a new Auction.
2. Place a bid - make sure there are no errors.
3. Place another bid - make sure there are no errors.
4. Attempt to view the Bid Product page (/products/bid-for-xxxxx) - It should redirect to the Auction page.
